src/tools/genpolicy/rules.rego

```diff
@@ -85,6 +90,38 @@ CreateContainerRequest {
     print("CreateContainerRequest: true")
 }
 
+allow_create_container_input {
+    print("allow_create_container_input: input =", input)
+
+    count(input.shared_mounts) == 0
+    is_null(input.string_user)
+
+    i_oci := input.OCI
+    is_null(i_oci.Hooks)
+    is_null(i_oci.Linux.Seccomp)
+    is_null(i_oci.Solaris)
+    is_null(i_oci.Windows)
+
+    i_linux := i_oci.Linux
+    count(i_linux.GIDMappings) == 0
+    count(i_linux.MountLabel) == 0
+    count(i_linux.Resources.Devices) == 0
```

Comment:
As mentioned in https://github.com/kata-containers/kata-containers/pull/9706#discussion_r1624683480, I think this is going to break device plugins. I'm not too concerned about that yet, but that may change when people want to use GPUs or something :)

 I have some code with which I can verify that on Monday.

Reply:
Thanks! Sure, if we have broken use cases, let's fix them.

Reply:
@burgerdev, do you think we should merge this PR? Or should we wait for additional experiments?

Thanks!

Reply:
Sorry for the delay. Things I verified to keep working:

* Adding a character device via device plugin
* Adding a block device via CSI

None of these sets `i_linux.Resources.Devices`, no idea how I arrived at that expectation. Anyway, please go ahead with the merge!

Reply:
No worries, thanks for helping out!